sample-geovelocity/siddhi-execution-geovelocity/src/main/java/org/wso2/extension/siddhi/execution/geovelocity/GetLoginBehaviourRisk.java

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wso2.extension.siddhi.execution.geovelocity;
+
+import org.wso2.extension.siddhi.execution.geovelocity.api.GeoVelocityData;
+import org.wso2.extension.siddhi.execution.geovelocity.api.GeoVelocityDataResolver;
+import org.wso2.extension.siddhi.execution.geovelocity.internal.exception.GeoVelocityException;
+import org.wso2.extension.siddhi.execution.geovelocity.internal.impl.GeoVelocityDataResolverHolder;
+import org.wso2.siddhi.annotation.Example;
+import org.wso2.siddhi.annotation.Extension;
+import org.wso2.siddhi.annotation.Parameter;
+import org.wso2.siddhi.annotation.ReturnAttribute;
+import org.wso2.siddhi.annotation.util.DataType;
+import org.wso2.siddhi.core.config.SiddhiAppContext;
+import org.wso2.siddhi.core.executor.ExpressionExecutor;
+import org.wso2.siddhi.core.executor.function.FunctionExecutor;
+import org.wso2.siddhi.core.util.config.ConfigReader;
+import org.wso2.siddhi.query.api.definition.Attribute;
+import org.wso2.siddhi.query.api.exception.SiddhiAppValidationException;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * This class is to get the risk of login based on
+ * the user's login behaviour.
```

Comment:
This can be in one line

Reply:
Fixed in 16a9ffcc4a35e7a36d08a524d44fba74b655e12a